docs/Other/Translating.md

```diff
@@ -47,6 +47,18 @@ Some entries have line breaks expressed as `\n`: Your translation can and in mos
 
 Chinese tutorial: 如果你是中国人，那么恭喜你运气不错！这里有Unciv中文开发者们专门为中文翻译工作者准备的（十分详尽）教程视频。：[(Video On Bilibili)](https://www.bilibili.com/video/BV1pY4y1u7WH/)
 
+## Adding a new language
+
+If any of the following steps are beyond your skillset, ask for help. All but the first two steps can be postponed.
+
+- You'll need to create a new file ('Create a new file' to the right of the folder name in the UI), and copy into it the contents of template.properties
+- For automatic language processing for a release (e.g. adding new templates) there needs to exist a line in [completionPercentages.properties](https://github.com/yairm210/Unciv/tree/master/android/assets/jsons/translations/completionPercentages.properties). Location and number do not matter, what matters is that the language name left of the ` = ` corresponds exactly with your new language file name, case-sensitive, without extension.
```

Comment:
Just curious why is this file not updated automatically.

Reply:
Just a random thought. If it was me, I would probably do something like this.
1. Count characters in a key. Keep track of sum.
2. If key has a value. Then increment another counter with the char count of the key.
3. In the end completion percentage would be `another counter / sum counter`.